lwt/dns_server.mli

```diff
@@ -33,6 +33,9 @@ end
 
 type 'a processor = (module PROCESSOR with type context = 'a)
 
+(** [compose process backup_process] takes two packet processes, returning one
+	which applies the second process only the first returns ServFail or NXDomain *)
+val compose: Dns.Packet.t process -> Dns.Packet.t process -> Dns.Packet.t process
```

Comment:
I think the doc string could be improved. It's usually better to describe the result, in this case the resulting process, in term of the variables you mention. IIUC what this does I would tentatively write it this way: 
```ocaml
(** [compose process backup_process] is [process] unless it returns {!ServFail} or {!NXDomain} in which case it becomes [backup_process]. *)
```